cli/setup.go

```diff
@@ -92,25 +92,21 @@ const (
 	demoRetryPoll         = 30
 	demoUpdatePoll        = 5
 	demoServerCertificate = "/usr/share/doc/mender-client/examples/demo.crt"
-	menderProfessionalURL = "https://hosted.mender.io"
+	hostedMenderURL       = "https://hosted.mender.io"
 
 	// Prompt constants
 	promptWizard = "Mender Client Setup\n" +
 		"===================\n\n" +
 		"Setting up the Mender client daemon: the daemon will " +
```

Comment:
As we have removed the "daemon" term below, I suggest we remove it here also. I think it is enough with "The Mender client will regularly poll..."

Reply:
I'll stick with ..."The client will regularly poll..." No need to repeat Mender everywhere :)